### PR TITLE
Fix/centrifugo reconnect

### PR DESCRIPTION
**Pull request by vladimir-8**

### CODE REVIEW

File: src/redux/middlewares/CentrifugoMiddleware/index.js [hunk 1]
```diff
@@ -16,81 +15,137 @@ import {
...
-    if (!isCentrifugoAction(action)) {
       return next(action);
     }
 
-    const state = getState();
-
     if (
-      !selectIsAuthenticated(state) ||
-      !selectHttpClientHasCredentials(state)
+      action.type === disconnect.type ||
```
Comment (Atala) [inline src/redux/middlewares/CentrifugoMiddleware/index.js:131]: shouldnt it be `&&` ?

File: src/redux/middlewares/CentrifugoMiddleware/actions.js [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
-export const CONNECTED = '@centrifugo/CONNECTED';
-export const DISCONNECTED = '@centrifugo/DISCONNECTED';
 
-export const connect = createAction(CONNECT);
+export const connect = createAction('@centrifugo/CONNECT');
+export const disconnect = createAction('@centrifugo/DISCONNECT');
 
-export const connected = createAction(CONNECTED);
-export const disconnected = createAction(DISCONNECTED);
+export const connected = createAction('@centrifugo/CONNECTED');
```
Comment (Atala) [inline src/redux/middlewares/CentrifugoMiddleware/actions.js:9]: should it be renamed `centrifugoConnected` for more clarity? (and other the same)

File: src/redux/middlewares/CentrifugoMiddleware/index.js (hunk 1 above)
Comment (vladimir-8) [reply src/redux/middlewares/CentrifugoMiddleware/index.js:131]: No, that's correct. We want to proceed only if both are true. To be honest, I think the second check is redundant, but I don't want to experiment with it now 🙈

File: src/redux/middlewares/CentrifugoMiddleware/actions.js (hunk 2 above)
Comment (vladimir-8) [reply src/redux/middlewares/CentrifugoMiddleware/actions.js]: Done

File: src/redux/middlewares/CentrifugoMiddleware/index.js (hunk 1 above)
Comment (Atala) [reply src/redux/middlewares/CentrifugoMiddleware/index.js]: that's why i am wondering it should be AND here? instead of OR

Comment (vladimir-8) [reply src/redux/middlewares/CentrifugoMiddleware/index.js]: The check is if NOT (`selectIsAuthenticated` AND `selectHttpClientHasCredentials`), which is the same as (NOT `selectIsAuthenticated`) OR (NOT `selectHttpClientHasCredentials`)

Comment (Atala) [reply src/redux/middlewares/CentrifugoMiddleware/index.js]: you are speaking about the removed code no?

**Review by Atala — APPROVED** [review]
tested OK with the following
- as restaurant, set airplane mode and the push connection comes back alright after reconnect 
- as restaurant, set lock screen and the push connection comes back alright after reconnect
- as rider, set airplane mode and the push connection comes back alright after reconnect

do you want to deploy to everybody or test in a few restaurants? we could use restaurant owner emails for that?

**Pull request merged**